server/models/procedures/stock.sql

```diff
@@ -650,9 +650,9 @@ BEGIN
 
           WHERE i.uuid = _inventory_uuid AND m.depot_uuid = _depot_uuid
              AND  (
-              ((m.start_date >= _start_date) AND  (m.end_date <= _end_date)) OR
-              ((m.start_date >= _start_date) AND  (m.end_date >= _end_date)) OR
-              ((m.start_date <= _start_date) AND  (m.end_date <= _end_date))
+              DATE(m.start_date) >= DATE(_start_date) AND DATE(m.end_date) <= DATE(_end_date)
```

Comment:
@lomamech has more experience here.  I know there were some bugs fixed in the logic here and I'm curious if this undoes them.